src/js/binary.js

```diff
@@ -29,6 +29,109 @@ export function navMenuListener() {
     });
 }
 
+export function selectDropdown(selector, has_label) {
+    if (!selector) return;
+
```

Comment:
how about assigning `const $selector = $(selector);` somewhere here so we don't have to keep fetching it using jquery?